deltacat/io/memcached_object_store.py

```diff
@@ -73,10 +69,15 @@ def put(self, obj: object, *args, **kwargs) -> Any:
         ref = self._create_ref(uid, create_ref_ip)
         client = self._get_client_by_ip(create_ref_ip)
 
-        if client.set(uid.__str__(), serialized, noreply=self.noreply):
-            return ref
-        else:
-            raise RuntimeError("Unable to write to cache")
+        try:
+            if client.set(uid.__str__(), serialized, noreply=self.noreply):
+                return ref
+            else:
+                raise RuntimeError("Unable to write to cache")
```

Comment:
Minor: Can we log the ref here as well to record the node IP for the failed cache set?

Reply:
yeah sure.